source/runtime/Rendering/Renderer.cpp

```diff
@@ -580,6 +591,16 @@ namespace spartan
         m_cb_frame_cpu.gamma               = GetOption<float>(Renderer_Option::Gamma);
         m_cb_frame_cpu.camera_exposure     = World::GetCamera() ? World::GetCamera()->GetExposure() : 1.0f;
 
+        // cloud/weather parameters
```

Comment:
REDO: renderer options will get moved in their own file and class (RendererOptionsPool.h)